.travis.yml

```diff
@@ -8,7 +8,7 @@ cache:
 
 matrix:
   include:
-    - php: 5.5
```

Comment:
5.5 should be here because we want to support that version also for the coming releases.